### PR TITLE
Methodology update

### PR DESCRIPTION
**Pull request by alice-hannah**
Hi Tina,

As I've mentioned, there are 22 care home locations with records in SMR01 - the new methodology excludes this activity from the hospital LOS so it is included as community activity. 

`01_create-basefile.R` now uses the new methodology and I have created another script `02_old-method.R` to retain the old way of calculating LOS and match this on to the basefile to allow for comparison. The `02_old-method.R` script should be exactly the same as the previous `01_create-basefile.R` script.

I've also added code for a comparison line chart in the `03_create-figures.R` script, that will eventually be included in the appendix of the report. I've not made any changes to the markdown document yet as I know I still have changes to make in the alice branch first.

You'll also notice that I've had to renumber the scripts to accommodate the new one - this _may_ cause a merge conflict but think it'll be straightforward to fix if it does.

Hope that's all ok. Would be much appreciated if you have time to review the new method code. Happy to discuss further tomorrow.

Alice 😄

### CODE REVIEW

**Comment by Tina815** [comment]
No problem Alice. Thanks for letting me know. I will take a look this week!

File: code/03_create-figures.R [hunk 1]
```diff
@@ -359,4 +359,54 @@ ggsave(here("markdown", "figures", "figure-a1-4.png"),
...
+  theme(panel.background = element_blank(),
+        panel.grid.major.x = element_blank(),
+        panel.grid.major.y = element_blank(),
+        axis.title.x = element_text(size = 8, face = "bold"),
+        axis.title.y = element_text(size = 8, face = "bold"),
+        axis.text = element_text(size = 8),
+        axis.text.x = element_text(angle = 90),
+        legend.title = element_blank(),
+        strip.text = element_text(size = 8, hjust = 1),
+        strip.background = element_blank()) +
```
Comment (Tina815) [inline code/03_create-figures.R]: I think we can remove these two lines as they are for facet plots as I know? I tried to remove them and the plot looks fine.

**Comment by Tina815** [comment]
Hi Alice, I have reviewed the new method and things should be fine! Just left 1 comment for the plot. After you have a look, I am happy to approve and merge it.

**Comment by alice-hannah** [comment]
Thanks Tina - should be ready to merge now :smile:

**Review by Tina815 — APPROVED** [review]

**Pull request merged**